sdk/keyvault/azure-security-keyvault-keys/test/ut/key_client_create_test_live.cpp

```diff
@@ -1,16 +1,28 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-#include "gtest/gtest.h"
-
+#include "../../azure-security-attestation/src/private/crypto/inc/crypto.hpp"
 #include "key_client_base_test.hpp"
 
+#include "private/key_constants.hpp"
+#include "private/key_serializers.hpp"
+#include "test_consts.hpp"
+#include "gtest/gtest.h"
+#include <azure/attestation.hpp>
+#include <azure/attestation/attestation_client_options.hpp>
+#include <azure/core/base64.hpp>
+#include <azure/core/internal/json/json.hpp>
 #include <azure/keyvault/keys.hpp>
 #include <private/key_constants.hpp>
-
 #include <string>
 
+using namespace Azure::Core::_internal;
+using namespace Azure::Security::KeyVault::Keys;
 using namespace Azure::Security::KeyVault::Keys::Test;
+using namespace Azure::Security::Attestation;
```

Comment:
Why are we including attestation headers and namespaces in keyvault live tests?

We shouldn't have one SDK (source or test) take a dependency on another. It seems odd to me, but maybe I am missing something.

Reply:
IIRC, KV's Secure Key Release requires using the attestation service. If we want a live test for SKR, you need to use the attestation service.